neon_hana/mq_websocket_api.py

```diff
@@ -33,7 +33,7 @@
 from neon_iris.client import NeonAIClient
 from ovos_bus_client.message import Message
 from threading import RLock
-from ovos_utils import LOG
+from ovos_utils.log import LOG
```

Comment:
Breaking change?

Reply:
Shouldn't be. `LOG` is and has been defined in `ovos_utils.log`, but there's a top-level package import. So, both of these are valid but I try to import from where things are defined so its easier to track down if you don't have it loaded in an IDE that resolves the reference for you